operations/finance.md

```diff
@@ -440,17 +469,90 @@ For most amounts, precision of values depends on `TaxPrecision` of [Enterprise](
 | `Code` | number | required | Code corresponding to tax type. |
 | `Value` | number | required | Amount of tax applied. |
 
-#### Credit card transaction
+#### Revenue type
+
+* `Service`
+* `Product`
+* `Additional`
+
+#### Order item data
```

Comment:
Just note that this is called OrderItemType in code. We might want to keep the naming same

Reply:
`Order item data` isn't call anything in code as that's the coproduct. You're right that `Order item data discriminator` is actually ``OrderItemType` in code, but this is consistent with rest of discriminator usage in documentation.